packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md

```diff
@@ -26,6 +26,314 @@ The MyAccount API has the following operations:
 * [Get my User Profile Schema](#get-my-user-profile-schema)
 * [Get my User Profile](#get-my-user-profile)
 * [Update my User Profile](#update-my-user-profile)
+* [Add My Email](#add-my-email)
+* [Get My Email](#get-my-email)
+* [Get My Emails](#get-my-emails)
+* [Delete My Email](#delete-my-email)
+
+
+### Add My Email
+
+<ApiOperation method="post" url="/idp/myaccount/emails" />
+
+Create an UNVERIFIED status, PRIMARY or SECONDARY email to user's MyAccount setting.
```

Comment:
Is `Create a PRIMARY or SECONDARY email in UNVERIFIED status to user's MyAccount setting` better?

Reply:
yeah sounds good